packages/react-percy/src/args.js

```diff
@@ -5,6 +5,11 @@ export const options = {
     alias: 'c',
     description: 'Path to the webpack config file',
   },
+  debug: {
+    description: 'Output Percy snapshots to disk for local debugging',
```

Comment:
Also maybe note that nothing is sent to percy in this case? That could be very confusing, might want it in the output text as well.

Reply:
Will add that to both places

Reply:
![image](https://user-images.githubusercontent.com/733696/30326457-92ad8c4e-9796-11e7-8e9f-dd0b141bf9f8.png)